### ISSUE TITLE
Replace "Cancel" with "Pause" and "Resume"

### ISSUE THREAD

**Issue by camsteffen** [camsteffen/polite#6]
In the notification, replace the "Cancel" button with "Pause". Then the notification will stay in place and the user can click "Resume".

**Comment by buxm** [comment]
Don't replace Cancel, but ADD to Cancel also Pause/Resume

**Comment by camsteffen** [comment]
> Don't replace Cancel, but ADD to Cancel also Pause/Resume

@buxm why?

**Comment by buxm** [comment]
So that I can still cancel and dismiss the notification (same as it currently works) if I don't need to silence the phone during the meeting (e.g. the meeting was cancelled last minute, nobody showed up, the meeting ended well before its deadline etc.)

**Comment by camsteffen** [comment]
I can make the notification dismiss-able when it is paused.

**Comment by jscottee** [comment]
I agree, it is nice just to have an option to cancel if I'm done early with a meeting.

**Comment by buxm** [comment]
I agree, that would be best... But either way is fine.
